src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +17,30 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> arr = new ArrayList<>();
+
+        File file = new File(fileName);
```

Comment:
try to create a solution without this variable

Reply:
Which one? Without File or without list?